alembic/ddl/base.py

```diff
@@ -175,7 +175,7 @@ def __init__(
         self.comment = comment
 
 
-@compiles(RenameTable)  # type: ignore[misc]
```

Comment:
oh....OK dont do unrelated typing changes here, if CI is not passing due to new mypy versions we would want to commit a separate fix for just typing issues.   we will do this in gerrit.  dont worry about jenkins not fully passing right now